bundles/org.openhab.binding.chatgpt/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -19,6 +19,26 @@
 				<label>API Key</label>
 				<description>API key to access the account</description>
 			</parameter>
+			<parameter name="apiUrl" type="text" required="false">
```

Comment:
Please add the new entries in chatgpt.properties file (just regenerate it) and add the 2 new configuration parameters in README.